easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb

```diff
@@ -78,9 +78,6 @@ exts_default_options = {
 # !! order of packages is important !!
 # packages updated on 17th March 2023
 exts_list = [
-    # include packages that are part of the base installation of R,
-    # both to make sure they are available (via sanity check),
-    # and to be able to pass the check for required dependencies when installing extensions in parallel
```

Comment:
@verdurin Let's retain these comments?

Reply:
Restored.

Reply:
I guess it's the `checksum` script that stripped them.

Reply:
Yeah, it's `--inject-checksums`, see also https://github.com/easybuilders/easybuild-framework/issues/3269